src/fetching/putEventResponse.ts

```diff
@@ -1,26 +1,20 @@
 import apiUrl from '../utils/apiUrl';
-
 import { ZetkinEventResponse, ZetkinMembership } from '../types/zetkin';
 
-function defaultFetch(path : string, init? : RequestInit) {
-    const url = apiUrl(path);
-    return fetch(url, init);
-}
-
 interface MutationVariables {
     eventId: number;
     orgId: number;
 }
 
-export default function putEventResponse(fetch = defaultFetch) {
+export default function putEventResponse() {
```

Comment:
Why return a function in this function? It should not be necessary to wrap the actual operations of this function in an inner function which is then returned. Just run the code in the body of `putEventResponse` and pass around `putEventResponse` instead of passing around the return value.

Reply:
This is a side-effect of the refactoring for server side rendering. But since neither the put or delete functions are run server side I'll revert these back to their previous state (same as you describe).